README.md

```diff
@@ -109,7 +109,9 @@ that improve the experience for consumers on these platforms.
 | Android   | arm32       |
 | iOS       | aarch64     |
 | Linux     | arm32       |
-| Linux     | Loongarch64 | 
+| Linux     | loongarch64 |
+| Linux     | risc-v64    |
```

Comment:
Should this go up in supported platforms now that it's in the CI?

Reply:
I think the standard for a platform being listed in the previous section is a little higher than just being in the CI.  But this should be discussed further.